tests/tracer/test_logger.py

```diff
@@ -133,14 +136,19 @@ def test_logger_init(self):
         self.assertIsNone(log.parent)
 
         # Override rate limit from environment variable
-        with self.override_env(dict(DD_LOGGING_RATE_LIMIT="10")):
+        with self.override_env(dict(DD_TRACE_LOGGING_RATE="10")):
             log = DDLogger("test.logger")
             self.assertEqual(log.rate_limit, 10)
 
         # Set specific log level
         log = DDLogger("test.logger", level=logging.DEBUG)
         self.assertEqual(log.level, logging.DEBUG)
 
+    def test_logger_deprecated_rate_limit(self):
```

Comment:
Do you mind adding a test for if both are set that we take the value from `DD_TRACE_LOGGING_RATE` over `DD_LOGGING_RATE_LIMIT`?